front/src/applications/stdcm/components/StdcmResults/StdcmDebugResults.tsx

```diff
@@ -54,7 +54,7 @@ const StdcmDebugResults = ({ simulationOutputs }: StdcmDebugResultsProps) => {
     <div className="stdcm-debug-results">
       {projectedData &&
         projectedData?.spaceTimeData.length > 0 &&
-        pathProperties.manchetteOperationalPoints && (
+        pathProperties?.manchetteOperationalPoints && (
```

Comment:
Hum you don't need this ?
```suggestion
        pathProperties.manchetteOperationalPoints && (
```